index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>Taskmanager</title>
+        <link rel="stylesheet" href="css/style.css" />
+    </head>
+    <body>
+        <h2>TODO list</h2>
+        <div class="main_list">
+            <table>
+                <tr>
+                    <td>
+                        New task:
+                    </td>
+                    <td>
+                        <form action="">
+                            <input type="text" name="" id="" placeholder="Insert your task here">
+                            <input type="submit" value="CREATE" class="button">
+                        </form>
+                    </td>
+                </tr>
+            </table>
+        
+        
+        <br>
+        <span>My tasks:</span>
+        <hr>
+        <div>
+            <ul>
+                <li>
+                    піся
+                </li>
+                <li>
+                    кака
+                </li>
+                <li>
+                    попа
+                </li>
+            </ul>
+        </div>
+    </div>
```

Comment:
Please fix indentation. Make sure that the tag opens and ends at the same level.

Reply:
Fixed
Aligned the code with Shift + Alt + F